Library/Homebrew/tap.rb

```diff
@@ -143,14 +145,37 @@ def official?
     user == "Homebrew"
   end
 
+  # @private
+  def read_and_set_private_config
```

Comment:
It's just occurred to me should we just inline this method in `private?`. It seems there is no point to create a new method here.